frontend/src/store/slice/Run/RunSelectors.ts

```diff
@@ -41,9 +41,9 @@ const selectForceRunList = (state: RootState) => {
   return nodes
     .filter(isAlgorithmNodeData)
     .filter((node) => {
-      const isUpdated = selectAlgorithmIsUpdated(node.id)(state)
+      const originalValue = selectAlgorithmIsUpdated(node.id)(state)
       const status = selectPipelineNodeResultStatus(node.id)(state)
-      return isUpdated || status === NODE_RESULT_STATUS.ERROR
+      return originalValue || status === NODE_RESULT_STATUS.ERROR
```

Comment:
Could you rename the variable `originalValue` to something like `isUpdated` ?

Reply:
@ReiHashimoto i fixed it, pls checkit again